include/ziti/ziti_tunneler.h

```diff
@@ -59,10 +59,19 @@ typedef struct tunneler_sdk_options_s {
     ziti_sdk_dial_cb   ziti_dial;
     ziti_sdk_close_cb  ziti_close;
     ziti_sdk_write_cb  ziti_write;
+    const char *ip_range;
```

Comment:
Would this be better named "autoip_range" or something to indicate that it's used for hostname-addressed services? Mainly I think "ip_range" might be confusing when we support network services, and also this field won't be used at all for IP-addressed services.

Reply:
it is not used anywhere, the setting will be passed to global DNS mapper instead